regression-test/pipeline/common/check-pr-if-need-run-build.sh

```diff
@@ -87,7 +87,9 @@ need_run_fe_ut() {
     for af in ${all_files}; do
         if [[ "${af}" == 'fe'* ]] ||
             [[ "${af}" == 'fe_plugins'* ]] ||
-            [[ "${af}" == 'bin/start_fe.sh' ]] ||
+            [[ "${af}" == 'fe_plugins'* ]] ||
```

Comment:
1. duplicate check fe_plugins
2. if start_fe.sh is changed, no need to run fe ut, org check is need?
3. sql-manual change need run fe ut is ok